ale_linters/cpp/clangtidy.vim

```diff
@@ -0,0 +1,15 @@
+" Author: vdeurzen <tim@kompiler.org>
+" Description: clang-tidy linter for cpp files
+
+" Set this option to change the clang-tidy options for warnings for C.
+if !exists('g:ale_cpp_clangtidy_options')
+    let g:ale_cpp_clangtidy_options = '-std=c++14 -Wall'
+endif
+
+call ale#linter#Define('cpp', {
+\   'name': 'clangtidy',
+\   'output_stream': 'stdout',
+\   'executable': 'clang-tidy',
+\   'command': g:ale#util#stdin_wrapper . ' -- ' . g:ale_cpp_clangtidy_options,
```

Comment:
Interestingly it was actually working with the given code. I'm now looking at the `stdin-wrapper` script but from the way that seems to work it will put the filename at the end. According to the clang-tidy documentation the filename has to come _before_ the `--`. Can I achieve that behavior with the given set-up?